--- react.js.orig	2014-08-01 23:24:56.000000000 +1000
+++ react.js	2014-08-01 23:25:11.000000000 +1000
@@ -15839,10 +15839,6 @@
  */
 function instantiateReactComponent(descriptor) {
   if ("production" !== "development") {
-    ("production" !== "development" ? warning(
-      isValidComponentDescriptor(descriptor),
-      'Only React Components are valid for mounting.'
-    ) : null);
     // We use the clone of a composite component instead of the original
     // instance. This allows us to warn you if you're are accessing the wrong
     // instance.
